assets/hello_world.c

```diff
@@ -0,0 +1,7 @@
+// This is a file use to compile some of the binaries in this
```

Comment:
I'm just noticing we have both assets and `tests/bin/` but whatever, not your fault